actions/templates.go

```diff
@@ -201,41 +187,37 @@ func PutTemplate(c *gin.Context) {
 
 	name := c.Param("name")
 
-	store, err := templates.NewSesTemplateStore(keys.AccessKey, keys.SecretKey, keys.Region)
+	templateInput := &entities.Template{
+		UserID:      u.ID,
+		Name:        name,
+		TextPart:    body.Content,
+		HTMLPart:    body.Content,
+		SubjectPart: body.Subject,
+	}
+
+	// create new template and try to parse html part
+	_, err := template.New("new").Parse(templateInput.HTMLPart)
```

Comment:
samce here

Reply:
@gudgl Yes, let's first find another way to validate.